### ISSUE TITLE
ESLint v4: indent wants too much indentation on last line of function params

### ISSUE THREAD

**Issue by karlhorky** [gajus/eslint-plugin-flowtype#263]
The last line of function parameters is indented too far when using a type and destructuring.

This happens with the following current versions of packages:

```sh
➜  cat node_modules/eslint-plugin-flowtype/package.json | grep version
  "version": "2.35.0"
➜  cat node_modules/eslint/package.json | grep version 
  "version": "4.3.0",
➜  cat node_modules/babel-eslint/package.json | grep version
  "version": "7.2.3"
➜  cat .eslintrc.js
module.exports = {
  extends: ['plugin:flowtype/recommended'],
  plugins: ['flowtype'],
  rules: {
    indent: ['error', 2], 
  },
};
➜  cat fn.js               
// @flow
function fn({
  key,
}: FnType) {}
➜  yarn lint -- --fix fn.js
yarn lint v0.27.5
$ eslint "--fix" "fn.js"
Done in 0.81s.
➜  cat fn.js
// @flow
function fn({
  key,
  }: FnType) {}  # <-- Indented too far
```

This does not happen with ESLint version `3.19.0`.

**Comment by not-an-aardvark** [comment]
This is fixed in ESLint 4.4.0.

**Comment by karlhorky** [comment]
> This is fixed in ESLint 4.4.0.

Thanks, confirmed. Closing this issue.
